src/Services/JobErrorHandler.php

```diff
@@ -25,14 +25,9 @@ public function clear()
     public function handleError($errno, $errstr, $errfile, $errline)
     {
         if (error_reporting()) {
-            // Don't throw E_DEPRECATED in PHP 5.3+
-            if (defined('E_DEPRECATED')) {
-                if ($errno == E_DEPRECATED || $errno == E_USER_DEPRECATED) {
-                    return;
-                }
-            }
-
             switch ($errno) {
+                case E_DEPRECATED:
+                case E_USER_DEPRECATED:
```

Comment:
Not really related but noticed it and seemed sensible to update this while I'm here, since `E_DEPRECATED` will always be defined in supported PHP versions now.